frontend/src/scenes/notebooks/NotebookMenu.tsx

```diff
@@ -0,0 +1,54 @@
+import { useActions, useValues } from 'kea'
```

Comment:
Late follow up to my review but I wonder if we should also add this to the popover
https://github.com/PostHog/posthog/blob/08cc576703694f1bcb277ed9a54535a5e71ccc7d/frontend/src/scenes/notebooks/NotebookPanel/NotebookPopover.tsx#L62-L69
Might be ok given very few people will see non-3000 notebooks

Reply:
Nah I'm happy for that view to get forgotten about 😅